packages/explat-client/src/internal/test-common.ts

```diff
@@ -12,3 +12,6 @@ export const validFallbackExperimentAssignment = {
 	ttl: 60,
 	isFallbackExperimentAssignment: true,
 };
+
+export const delayedValue = < T >( value: T, delayMilliseconds: number ): Promise< T > =>
```

Comment:
I see that delayMilliseconds is always passed in as 1 in the new tests. Maybe make it optional and default to 1 so we can simply call `delayedValue( value )`?

Reply:
I think it is better to be explicit here, also for some tests I use `0`.

Reply:
Hmm... It's a bit hard to see what the 1 is referring to when reading the calling code. Maybe create constants and pass them in explicitly then? Something like this:

```javascript
const NO_DELAY = 0;
const MIN_DELAY = 1;

delayedValue( <value>, MIN_DELAY );
```

Reply:
I'm using `ZERO_DELAY` and `ONE_DELAY` as I think `NO` and `MIN` are misleading:
- `NO_DELAY` won't be run instantly, instead it will be behaviourally equivalent to `setTimeout(f, 0)`
- `MIN_DELAY`: 0 is the min delay not 1.

Reply:
Also I am not using it in `timing.ts` as many other functions are using time and these values need to be easily seen and compared - especially with mock timers now being used.